### DISCUSSION TITLE
Bromite Browser pings bromite.org on launch, why?

### DISCUSSION THREAD

**Discussion by LowTalkDev** [bromite/bromite#1470] [Q&A]
As the title states bromite pings bromite.org even with adblock and any other relevant settings disabled, my bromite doesnt ping anything except bromite.org on launch of the app. Because of the apps field of interest I believe it reasonable to ask why it does so? Its stated nowhere I can find and personally I feel this should be stated clearly somewhere, what is the dev teams opinion?

**Comment by LowTalkDev** [comment]
Found this https://github.com/bromite/bromite/discussions/1195

But i dont have any setting for adblock on to be pinging for that reason so why is it still doing it?

**Reply by csagan5** [reply]
If you have the website on the NTP, it will load the favicon from bromite.org (as it does for every other website there).
